### PR TITLE
Knit GvsExtractCohortFromSampleNames.wdl into GvsCalculatePrecisionAndSensitivity.wdl [VS-1245]

### PR DESCRIPTION
**Pull request by rsasch**
- uses GvsExtractCohortFromSampleNames.wdl to generate VCFs for calculating P & S all in one WDL
- allows for use of an interval_list in P & S
- updates to docs
- Successful run here: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/2537ea7b-f635-4609-8fbb-7eaec41a6df8
- integration run, since I touched the bulk ingest and extract VCF WDLs: https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/f71941ba-f02f-4472-a04c-aedff24fdd14

### CODE REVIEW

File: scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl [hunk 1]
```diff
@@ -19,6 +19,7 @@ workflow GvsBulkIngestGenomes {
         # Begin GvsAssignIds
         String dataset_name
         String project_id
+        Boolean samples_are_controls = false
```
Comment (rsasch) [inline scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl:22]: added this in order to ingest control samples into my quickstart

Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl:22]: ooooh does this mean we can load controls via bulk ingest now?!?!?

File: scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl [hunk 2]
```diff
@@ -446,18 +396,10 @@ task EvaluateVcf {
...
-    for i in "${chromosomes[@]}"
-    do
-      echo "$i	0	300000000" >> chromosomes.to.eval.txt
-    done
-
     rtg format --output human_REF_SDF ~{ref_fasta}
 
     rtg vcfeval \
-      --bed-regions chromosomes.to.eval.txt \
+      --bed-regions ~{interval_list} \
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl:402]: so much better!

File: scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl [hunk 3]
```diff
@@ -161,62 +169,6 @@ workflow GvsCalculatePrecisionAndSensitivity {
...
-      if [ $VCF_CHR = $i ]; then
-        cp ${input_vcf} output/${output_vcf_name}
-        echo "Including ${input_vcf_path} as it is on $i."
-        break
-      else
-        touch output/${output_vcf_name}
-        echo "Skipping ${input_vcf_path} as it is not on $i."
-      fi
-    done
-  >>>
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl:207]: omg why was this a thing lol

Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl:207]: omg why was this a thing lol

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl:22]: In a separate run after creating a sample_set of control samples, yes.  Not sure if that does everything that https://broadworkbench.atlassian.net/browse/VS-1205 had in mind, tho.

**Review by koncheto-broad — APPROVED** [review]
LGTM.  Not sure I'm 100% happy about us completely getting rid of the ability to do a simplified P&S on an existing prepare table just to support this new method that is largely to satisfy Lee's curiosity, but if everybody else is cool with it then you get my thumb as well. I'm personally prefer to retain the ability to do both, but this is also acceptable.

File: scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl (hunk 3 above)
Comment (mcovarr) [reply scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl:207]: This was part of @gbggrant's work for VS-390

File: scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl [hunk 4]
```diff
@@ -13,19 +13,20 @@ workflow GvsExtractCohortFromSampleNames {
...
 
-    String query_project
     String gvs_project
     String gvs_dataset
     String call_set_identifier
-    String cohort_table_prefix
+    String cohort_table_prefix = call_set_identifier
+    String query_project = gvs_project
 
     # not using the defaults in GvsPrepareCallset because we're using pre created datasets defined by the caller
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl:22]: tbh I don't understand this comment but I'm wondering if it's still "correct" with these changes?

File: scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl [hunk 5]
```diff
@@ -95,21 +102,28 @@ workflow GvsExtractCohortFromSampleNames {
   # allow an interval list to be passed in, but default it to our standard one if no args are here
   File working_interval_list = select_first([interval_list, "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"])
 
+
+  #   if (!defined(git_hash) || !defined(variants_docker) || !defined(cloud_sdk_docker)) {
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl:106]:
```suggestion
```

File: scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl (hunk 1 above)
Comment (mcovarr) [reply scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl:22]: can we plz look at that ticket and update / close as appropriate based on the changes here?

**Review by mcovarr — COMMENTED** [review]
I share Aaron's concerns about losing the ability to do P&S on an existing prepare table. However I do like that this wraps an important WDL that we have not been running ourselves with a WDL that we actually do run from time to time, so that's a plus.

**Review by mcovarr — APPROVED** [review]

**Pull request merged**